node/core/pvf/src/host.rs

```diff
@@ -1419,6 +1450,29 @@ mod tests {
 			test.poll_and_recv_to_prepare_queue().await,
 			prepare::ToQueue::Enqueue { .. }
 		);
+
+		test.from_prepare_queue_tx
+			.send(prepare::FromQueue {
+				artifact_id: artifact_id(1),
+				result: Ok(Duration::default()),
+			})
+			.await
+			.unwrap();
+
+		// Preparation should have been retried and succeeded this time.
+		let result_tx_3 = assert_matches!(
+			test.poll_and_recv_to_execute_queue().await,
+			execute::ToQueue::Enqueue { result_tx, .. } => result_tx
+		);
+
+		// Send an error for the execution here, just so the result receiver gets something.
```

Comment:
This is the first look I've had at the PVF code. Cool stuff! 

Out of curiosity, why is it important that the result receiver gets something here?

Reply:
Cool! This seemed like an easy, bite-sized intro to the module. 🙂

> Out of curiosity, why is it important that the result receiver gets something here?

I wanted to test that the receiver was still operational, i.e. not dropped. I guess alternatively I could poll it and it should return `Ok(None)`, but it felt more complete to send something. And an `Ok` value was more complicated to send as it would require sending a worker. Does that answer your question? Let me know if I can clarify this, so it's more apparent to readers of the code.

Reply:
This is totally optional, but modifying the comment to something like "just so we can check the result receiver is still alive" would've been more clear to me.